src/validation.h

```diff
@@ -451,6 +451,19 @@ class Chainstate
     //! Manages the UTXO set, which is a reflection of the contents of `m_chain`.
     std::unique_ptr<CoinsViews> m_coins_views;
 
+    //! This toggle exists for use when doing background validation for UTXO
+    //! snapshots.
+    //!
+    //! In the expected case, it is set once the background validation chain reaches the
+    //! same height as the base of the snapshot and its UTXO set is found to hash to
+    //! the expected assumeutxo value. It signals that we should no longer connect
+    //! blocks to the background chainstate. When set on the background validation
+    //! chainstate, it signifies that we have fully validated the snapshot chainstate.
```

Comment:
In commit "validation: add CChainState::m_disabled and ChainMan::isUsable" (e07764318ad766ba27dfeafe1ca2345534f81dd5)

This last sentence is confusing because it is part of the "In the expected case" paragraph, but it is referring to an different unexpected case. It's also not very clear that this sentence is referring to a different chainstate object than the rest of the paragraph.

Would move this sentence to a separate paragraph and say something like "In the error case, if the background UTXO hash does not match the expected assumeutxo valid, m_disabled will be set to true on the snapshot chainstate (not the background validation chainstate) indicating that the snapshot chainstate is invalid."